scrapers/SmutPuppet.yml

```diff
@@ -0,0 +1,66 @@
+name: SmutPuppet
+sceneByURL:
+  - action: scrapeXPath
+    url: 
+      - blackandbig.com/update
+      - darksodomy.com/update
+      - dothewife.com/update
+      - dreamtranny.com/update
+      - genlez.com/update
+      - goldenslut.com/update
+      - grannyvsbbc.com/update
+      - jefssmodels.com/update
```

Comment:
Small typo here, should be `jeffsmodels.com/update`

Reply:
Fixed